docs/usage/release-notes-for-major-versions.md

```diff
@@ -7,6 +7,69 @@ The most recent versions are always at the top of the page.
 This is because recent versions may revert changes made in an older version.
 You also don't have to scroll to the bottom of the page to find the latest release notes.
 
+## Version 39
+
+### Breaking changes for 39
+
+#### New tools and config for Renovate's Docker images
+
+Renovate Docker images now use Node.js v22 as base, this was Node.js v20.
+
+##### Sidecar images
+
+Renovate's "sidecar" images now:
+
+- Use Ubuntu 24.04 as base, this was 20.04
+- Set the user ID to `12021`, this was `1001`
+
+##### Full images
+
+Renovate's `-full` images now uses:
+
+- Ubuntu 24.04
+- Python 3.13
+- Node.js v22
+
+The Renovate `-full` image defaults to [`binarySource=global`](https://docs.renovatebot.com/self-hosted-configuration/#binarysource).
+If you want to keep the old behavior, where Renovate dynamically installs the needed tools: set the environment variable `RENOVATE_BINARY_SOURCE` to `"install"`.
+
+Question do we need to update the `binarySource` docs?
+It still says:
+
+> Starting in v36, Renovate's default Docker image (previously referred to as the "slim" image) uses `binarySource=install` while the "full" Docker image uses `binarySource=global`. If you are running Renovate in an environment where runtime download and install of tools is not possible then you should use the "full" image.
+
+#### Prefer squash merges for automerges on GitHub
+
+When automerging, Renovate tries the "squash merge" method first.
+
+#### Branch names with multiple slashes
+
+Branch names with multiple forward slashes (`/`) will change if `branchNameStrict=true`.
+
+### Commentary for 39
+
+#### Prefer squash merge for automerges to allow signed commits on GitHub
+
+This info is for the GitHub platform only.
+
+When you sign your commits on GitHub, you'll want Renovate's commits to be signed too.
+If Renovate automerges its PRs with the **Rebase and Merge** option, then GitHub can't sign the commits.
+But GitHub can sign squash-merged commits for Renovate.
+This is why we changed the default behavior to try the squash merge first.
+
+Read the [GitHub Docs, Signature verification for rebase and merge](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#signature-verification-for-rebase-and-merge) to learn more.
+
+Note from HonkingGoose to maintainers, the [GitHub Docs, signature verification for bots](https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification#signature-verification-for-bots) have more info about signing commits as bot author.
+I don't know if we already do this, or if the method is better than relying on a specific kind of merge type?
```

Comment:
Are we already doing this? I wonder if this may help avoid things like relying on a specific merge method to keep Renovate's commits signed?

> ### Signature verification for bots
>
> Organizations and GitHub Apps that require commit signing can use bots to sign commits. If a commit or tag has a bot signature that is cryptographically verifiable, GitHub marks the commit or tag as verified.
>
> Signature verification for bots will only work if the request is verified and authenticated as the GitHub App or bot and contains no custom author information, custom committer information, and no custom signature information, such as Commits API.